src/components/Form_Components/Login/Login.js

```diff
@@ -30,7 +33,11 @@ const default_values = {
   }
 };
 const default_user = { email: "", password: "" };
+ga('create', 'UA-XXXXX-Y', 'auto', {
+  userId: state.user
+});
 
+ga('set', 'userId', state.user)
```

Comment:
Let's double check that this is working locally. Is this a global function or do we need to import/write this `ga` function?